sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CUserFlowCondition.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.autoconfigure.b2c;
+
+import org.springframework.boot.autoconfigure.condition.ConditionOutcome;
+import org.springframework.boot.autoconfigure.condition.SpringBootCondition;
+import org.springframework.boot.context.properties.bind.Binder;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+import org.springframework.util.CollectionUtils;
+
+/**
+ * Configuration condition for activating AAD B2C OAuth2 security configuration beans.
+ */
+public class AADB2CUserFlowCondition extends SpringBootCondition {
```

Comment:
Can we create a class called `AADBCConditions` and put all these conditions in that one?

Reply:
Good idea.